javascript/ql/src/Security/CWE-830/FunctionalityFromUntrustedDomain.ql

```diff
@@ -0,0 +1,18 @@
+/**
+ * @name Untrusted domain used in script or other content
+ * @description Use of a script or other content from an untrusted or compromised domain
```

Comment:
This doesn't follow the guidelines for query descriptions (see https://github.com/github/codeql/blob/main/docs/query-metadata-style-guide.md#query-descriptions-description).

Maybe this would work?

```suggestion
 * @description Using a resource from an untrusted or compromised domain makes your code vulnerable to receiving malicious code.
```